internal/stacks/stack.go

```diff
@@ -494,11 +510,25 @@ func (s *Stack) runFirstTimeSetup(spin *spinner.Spinner, verbose bool, options *
 	if err := docker.RunDockerComposeCommand(workingDir, verbose, verbose, "up", "-d"); err != nil {
 		return err
 	}
-	containerName := fmt.Sprintf("%s_firefly_core_%s_1", s.Name, s.Members[0].ID)
+
+	var containerName string
+	for _, member := range s.Members {
+		if !member.External {
+			containerName = fmt.Sprintf("%s_firefly_core_%s_1", s.Name, member.ID)
```

Comment:
Could break early out of this loop?

Reply:
Yes I think I meant to do that. Thanks, good catch.